rids/iocs/bad_ip_list.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Jigsaw Operations LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""IOC feed parser for lists of compromised host IPs (newline separated)."""
+
+
+from datetime import datetime
+import ipaddress
+
+from rids import rules
+
+
+class FeedParser:
+  """Parses newline-separated list of IP addresses into a RuleSet."""
+  def __init__(self, config):
+    self.source_url = config['url']
+    self.source_name = config.get('name', 'UNKNOWN')
+    self.msg = config.get('msg', 'contact with a potentially compromised host')
+
+  def ParseRules(self, ioc_data, ruleset):
+    timestamp = str(datetime.now())
```

Comment:
Can you make this a real `datetime` object? It's a pain to deal with timestamps otherwise.

Reply:
yeah, I can.  I probably would have as soon as adding the periodic re-fetch of IOCs, to calculate duration.

done